types/wnumb/index.d.ts

```diff
@@ -1,8 +1,11 @@
-// Type definitions for wnumb 1.0
+// Type definitions for wnumb 1.2.0
```

Comment:
this should read `1.2` I believe:
https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library